src/ui/Sort/Sort.ts

```diff
@@ -285,40 +374,20 @@ export class Sort extends Component {
   }
 
   private updateAccessibilityProperties() {
-    this.updateAccessibleSelectedState();
-    this.updateAccessibleLabel();
-  }
-
-  private updateAccessibleSelectedState() {
-    this.element.setAttribute('aria-pressed', this.isSelected().toString());
-  }
-
-  private updateAccessibleLabel() {
-    this.element.setAttribute('aria-label', this.getAccessibleLabel());
-  }
-
-  private getAccessibleLabel() {
-    return this.isToggle() ? this.getAccessibleLabelWithSort() : this.getAccessibleLabelWithoutSort();
+    this.sortButton.setAttribute('aria-checked', `${this.isSelected() && this.currentDirection === this.initialDirection}`);
+    if (this.canChangeDirection()) {
+      this.directionButton.setAttribute('aria-checked', `${this.isSelected() && this.currentDirection !== this.initialDirection}`);
```

Comment:
We can store the second half of the condition in a variable and reuse here and on line 377.